Preprocessing/OneHotEncoder.ecl

```diff
@@ -0,0 +1,83 @@
+/*##############################################################################
+## HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems.  All rights reserved.
+############################################################################## */
+
+IMPORT $.^ as ML_Core;
+IMPORT $.Types;
+NumericField := ML_Core.Types.NumericField;
+l_cFeatures := Types.OneHotEncoder.cFeatures;
+
+/**
+  * OneHotEncoder
```

Comment:
Need to indicate what this does.  For example:

Used to convert each of the designated categorical features to a binary (absent / present) value (i.e. oneHot) for use by algorithms that don't directly support categorical values.  Also can convert back from oneHot encoding to numerical category.  Each categorical field will produce additional features according to its cardinality.  
For example, if there are four possible categories, then the original feature will be replaced by four binary features.  These are placed after all of the non-categorical features in your dataset.